client/rest-client/src/main/java/org/infinispan/client/rest/RestClient.java

```diff
@@ -69,6 +69,18 @@
     */
    RestClientConfiguration getConfiguration();
 
+   CompletionStage<RestResponse> metrics();
```

Comment:
Maybe we can move this to a RestMetricsClient interface. Since this module is experimental, no need to worry about backwards-compatibility.

Reply:
Indeed. I realised only now that we'll never need an actual java client for metrics. Monitoring apps (and our console ) will use the rest endpoint straight over http. The only user for such an api would be our test suite. So I'll make a jira to extract a RestMetricsClient and put it in tests for now. We can promote it if ever needed.